src/configs.rs

```diff
@@ -20,22 +16,16 @@ pub(crate) struct Opts {
     /// Enabled Indexer for Explorer debug level of logs
     #[clap(long)]
     pub debug: bool,
-    /// AWS S3 bucket name to get the stream from
-    #[clap(long, env)]
-    pub s3_bucket_name: String,
-    /// AWS S3 bucket region
-    #[clap(long, env)]
-    pub s3_region_name: String,
```

Comment:
@roshaans that was your job in previous PR to drop unused args
It was confusing that we no longer use it and it's still possible to provide it